lib/commands/gesture.js

```diff
@@ -24,11 +24,15 @@ helpers.handleTap = async function (gesture) {
     params.y = options.y;
   }
 
-  let endpoint = `/tap/${util.hasValue(options.element) ? options.element : '0'}`;
+  let endpoint = '';
+  if (gesture.action === 'tap') {
+    endpoint = `/${gesture.action}/${util.hasValue(options.element) ? options.element : '0'}`;
```

Comment:
I would move the repeated part out, since it is the same in either case:

```js
let el = util.hasValue(options.element) ? options.element : '0';
if (gesture.action === 'tap') {
  endpoint = `/${gesture.action}/${el}`;
} // ...
```